ui/admin/tests/acceptance/auth-methods/list-test.js

```diff
@@ -69,13 +71,75 @@ module('Acceptance | auth-methods | list', function (hooks) {
       scope: instances.scopes.org,
       type: TYPE_AUTH_METHOD_OIDC,
     });
-    urls.globalScope = `/scopes/global/scopes`;
+    urls.globalScope = `/scopes/global`;
     urls.orgScope = `/scopes/${instances.scopes.org.id}`;
     urls.authMethods = `/scopes/${instances.scopes.org.id}/auth-methods`;
+    urls.globalAuthMethods = `${urls.globalScope}/auth-methods`;
+    urls.orgAuthMethods = `${urls.orgScope}/auth-methods`;
     urls.passwordAuthMethod = `${urls.authMethods}/${instances.passwordAuthMethod.id}`;
     urls.oidcAuthMethod = `${urls.authMethods}/${instances.oidcAuthMethod.id}`;
 
-    authenticateSession({});
+    authenticateSession({ username: 'admin' });
+  });
+
+  module('a11yAudit', function () {
+    test.each('auth-methods', ['light', 'dark'], async function (assert, data) {
+      assert.expect(0);
+      currentSession().set('data.theme', data);
+      await visit(urls.authMethods);
+
+      // open new dropdown
+      await click('.rose-layout-page-actions button');
+      await a11yAudit();
+
+      // open dropdown at last row
+      await click('td:last-child button');
+      await a11yAudit();
+
+      // open primary filter
+      await click('.hds-segmented-group div[name="primary"] div button');
+      await a11yAudit();
+
+      // check 'yes'
+      await click(
+        '.hds-segmented-group div[name="primary"] div:last-child input',
+      );
+      await a11yAudit();
+
+      // filter selected
+      await click(FILTER_DROPDOWN_SELECTOR('type'));
+      await click(`input[value="${TYPE_AUTH_METHOD_PASSWORD}"]`);
+      await click(FILTER_APPLY_BUTTON_SELECTOR);
+      await a11yAudit();
+
+      // "no results" message
+      await fillIn(
+        SEARCH_INPUT_SELECTOR,
+        'fake auth method that does not exist',
+      );
+      await a11yAudit();
+    });
+
+    test('API error', async function (assert) {
+      assert.expect(0);
+      this.server.get('/auth-methods', () => {
+        return new Response(
+          418,
+          {},
+          {
+            status: 418,
+            code: "I'm a teapot",
+            message: 'Ope, sorry about that!',
+          },
+        );
+      });
+
+      await visit(urls.globalAuthMethods);
+      await a11yAudit();
+
+      currentSession().set('data.theme', 'dark');
+      await a11yAudit();
+    });
   });
 
   test('users can navigate to auth methods with proper authorization', async function (assert) {
```

Comment:
None blocking comment: the group of tests which do not belong to the a11Audit aren't group by a `module` statement. Is this in purpose?
Following the pattern we have in the acceptance test, i.e: [users list-test](https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/tests/acceptance/users/list-test.js), [auth-methods delete-test](https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/tests/acceptance/auth-methods/delete-test.js#L21), we wrap them in a module, such as `'Acceptance | auth-methods | list'`.

Reply:
Thanks for pointing this out as looking at this is confusing. All the tests in this file are wrapped in a module named `'Acceptance | auth-methods | list'`. What I did was add another module within that just for the `a11yAudit` tests. The main reason for this is we get access to all of the work done in the `beforeEach`. I wonder if maybe I should move the whole `a11yAudit` module to the bottom of the parent test module to make this less confusing?